src/common/vfs.h

```diff
@@ -31,7 +31,6 @@ typedef QSharedPointer<Account> AccountPtr;
 class SyncJournalDb;
 class VfsPrivate;
 class SyncFileItem;
-typedef QSharedPointer<SyncFileItem> SyncFileItemPtr;
```

Comment:
(Putting a message on the first line of the commit, but i want to comment on the whole commit)

I do not understand the relation between the commit message and this commit.
It does not seem to touch directory, it is just some refactoring, is it not?

Reply:
The significant change that happens in "Vfs: Make files that end up in db placeholders" is that `convertToPlaceholder()` is called for each `updateMetadata()`. Previously it was called much more sparingly and that would cause trouble down the line.